build/arm-tools.mk

```diff
@@ -45,9 +45,14 @@ ASFLAGS += -mfloat-abi=hard -mfpu=fpv4-sp-d16
 LDFLAGS += -nostartfiles -Xlinker --gc-sections
 endif
 
+CPPFLAGS += -flto -ffat-lto-objects -DPARTICLE_COMPILE_LTO_FAT
+CONLYFLAGS += -flto -ffat-lto-objects -DPARTICLE_COMPILE_LTO_FAT
```

Comment:
A comment clarifying that we're not enabling LTO by default would be nice here. Maybe it's just me, but I didn't know about fat LTO objects.

Reply:
Added